appveyor.yml

```diff
@@ -0,0 +1,30 @@
+version: 3.5.0-xprivate-{build}
+pull_requests:
+  do_not_increment_build_number: true
```

Comment:
this setting is for PR builds only as opposed to on-every-commit dev builds. Copied from other ASP.NET project. Will evaluate it later when actual PR integration is in place.